src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -26,8 +31,16 @@
 #include "Parallel/Invoke.hpp"
 #include "Parallel/PhaseDependentActionList.hpp"
 #include "Parallel/Printf.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
 #include "ParallelAlgorithms/DiscontinuousGalerkin/InitializeDomain.hpp"
```

Comment:
Delete this line

Reply:
When I remove line 34, I get this compiler error: 
~~~
/home/geoffrey/Codes/spectre/spectre/src/Executables/ExportCoordinates/ExportCoordinates.hpp:183:16: error: no member named 'register_derived_classes_with_charm' in namespace 'Parallel'
    &Parallel::register_derived_classes_with_charm<TimeStepper>,
~~~

Reply:
I meant line 35, since you replaced the action with the one from `Evolution`

Reply:
O, I misunderstood! My eye went to the line I changed. Fixed and squashed @nilsleiffischer